beater/beater_test.go

```diff
@@ -280,3 +282,27 @@ func TestTransformConfig(t *testing.T) {
 	test(true, false, false)
 	test(true, true, true)
 }
+
+func TestStoreUsesRUMElasticsearchConfig(t *testing.T) {
```

Comment:
Can we add a basic unit test that shows the Fleet sourcemap store backend is used when Fleet config is defined?